mesonbuild/backend/xcodebackend.py

```diff
@@ -1121,7 +1121,7 @@ def generate_pbx_project(self, objects_dict):
         project_dict.add_item('buildStyles', style_arr)
         for name, idval in self.buildstylemap.items():
             style_arr.add_item(idval, name)
-        project_dict.add_item('compatibilityVersion', '"Xcode 3.2"')
+        project_dict.add_item('compatibilityVersion', '"Xcode 11.4"')
```

Comment:
Does this mean we're dropping support for old xcode versions? I don't necessarily object to the idea, but if so we should probably document it in the release notes.

Reply:
This was mainly so "Parallelize build for command-line builds" would have any effect whatsoever. Apart from that, most Xcode users are not using anything older than 11.4 anyway.

Reply:
I have a better idea for handling this, but that's for a separate PR. For now, dropping this commit.